plugin.audio.radiobrowser/addon.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<addon id="plugin.audio.radiobrowser" name="Radio-Browser.info" version="1.0.0" provider-name="sailor">
+<addon id="plugin.audio.radiobrowser" name="Radio-Browser.info" version="1.1.0" provider-name="sailor">
   <requires>
     <import addon="xbmc.python" version="2.24"/>
```

Comment:
> WARN: For jarvis it is advised to set xbmc.python version to 2.24.0

Reply:
It was a random failure, caused by some http request timeout.
Please change it to `2.24.0` again.